Detector/DetFCCeeCLD/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ target_link_libraries(DetFCCeeCLD DD4hep::DDCore)
 set(LIBRARY_OUTPUT_PATH ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
 
 add_test(NAME ConstructDetFCCeeCLD
-         COMMAND root -q -b ".github/scripts/load_detector.C(\"Detector/DetFCCeeCLD/compact/FCCee_o2_v02/FCCee_o2_v02.xml\", \"fccee_cld.root\")"
+         COMMAND root -q -b ".github/scripts/load_detector.C(\"Detector/DetFCCeeCLD/compact/FCCee_o2_v02/FCCee_o2_v02.xml\")"
```

Comment:
Hopefully the removal will not be necessary with https://github.com/HEP-FCC/FCCDetectors/pull/42 .

Reply:
thank you, I updated with the main